src/Kunstmaan/TranslatorBundle/Form/TranslationsFileUploadType.php

```diff
@@ -19,6 +19,9 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         $builder->add('file', FileType::class, [
             'required' => true,
             'label' => 'kuma_translator.form.upload_file_choose',
+            'constraints' => array(
```

Comment:
Can you use the short array syntax here? And add a trailing comma to the `new NotBlank()` line. Thanks!

Reply:
no problem :)  I see two types mixed in this file